### PR TITLE
account changes

### PR DESCRIPTION
**Pull request by sunnya97**
- External Actor controls Externally Controlled Account, not any Account
- Added Account to domain of hasAccountStorage
- Moved Account code hash to be a property of Account State, not Contract Account
- Renamed Storage to Account Storage

### CODE REVIEW

**Comment by webthethird** [comment]
Points 1, 2 and 4 seem like straightforward fixes

Would be good to get feedback from @jo-tud on the third point

Does the account code hash ever change once it has been uploaded to the EVM?

**Comment by sunnya97** [comment]
No, account code hash will never change.

**Comment by webthethird** [comment]
@sunnya97 Then can you explain your reasoning for moving 'accountCodeHash` to be a property of `Account State` rather than `Contract Account`? I see in the current definition of Account State that 
>  It is comprised on four pieces of information: nonce, balance, storage root and code hash
which would seem to justify the change. But intuitively the term Account State implies change to me, since nonce, balance and storage all have changing states. Does that make sense?

**Comment by webthethird** [comment]
Would be interested in hearing what @chronaeon thinks

File: EthOn.rdf [hunk 1]
```diff
@@ -774,7 +775,7 @@
     <owl:DatatypeProperty rdf:about="http://ethon.consensys.net/accountCodeHash">
         <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/AccountDataProperty"/>
         <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#FunctionalProperty"/>
-        <rdfs:domain rdf:resource="http://ethon.consensys.net/ContractAccount"/>
+        <rdfs:domain rdf:resource="http://ethon.consensys.net/Account"/>
```
Comment (jo-tud) [inline EthOn.rdf]: On third thought, even though this is not what is stated in the PR comment, it is reasonable and can be merged.

**Comment by jo-tud** [comment]
@webthethird The definition of AccountState really does differ from the ontological model atm. I changed it.

**Comment by webthethird** [comment]
Excellent, thanks for taking care of that. Makes sense to me!

**Pull request merged**